appveyor.yml

```diff
@@ -0,0 +1,9 @@
+environment:
+  matrix:
+        - jdk: "C:\\Program Files\\Java\\jdk1.8.0\\bin:"
+
+build_script:
+  # Build the compiled extension
+  - "gradlew.bat install check -Prunit=online -DwlpVersion=17.0.0.3 --stacktrace --info"
+
+test: off
```

Comment:
What automated tests does this turn off?  Are the tests running?

Reply:
The tests are running from our buildscript command. This is appveyor configuration where we could break apart our buildscript command and add the test part of the script here, but with no real benefit.